### PR TITLE
Update gems

### PR DESCRIPTION
**Pull request by jhass**
#3855

### CODE REVIEW

File: bin/cucumber [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env ruby
 begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
+  load File.expand_path('../spring', __FILE__)
```
Comment (diaspora-code-review) [inline bin/cucumber]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: bin/cucumber [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+rescue LoadError => e
+  raise unless e.message.include?('spring')
```
Comment (diaspora-code-review) [inline bin/cucumber]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: bin/rails [hunk 3]
```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env ruby
 begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
+  load File.expand_path('../spring', __FILE__)
+rescue LoadError => e
+  raise unless e.message.include?('spring')
 end
+# This command will automatically be run when you run "rails" with Rails 3 gems installed from the root of your application.
```
Comment (diaspora-code-review) [inline bin/rails]: Line is too long. [124/120]

File: bin/rails [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
+
 APP_PATH = File.expand_path('../../config/application',  __FILE__)
-require_relative '../config/boot'
+require File.expand_path('../../config/boot',  __FILE__)
```
Comment (diaspora-code-review) [inline bin/rails]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Comment (diaspora-code-review) [inline bin/rails]: Unnecessary spacing detected.

File: bin/spring [hunk 5]
```diff
@@ -1,18 +1,15 @@
...
 
-# This file loads spring without using Bundler, in order to be fast
-# It gets overwritten when you run the `spring binstub` command
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
```
Comment (diaspora-code-review) [inline bin/spring]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: bin/spring [hunk 6]
```diff
@@ -1,18 +1,15 @@
...
+  require 'bundler'
```
Comment (diaspora-code-review) [inline bin/spring]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: bin/spring [hunk 7]
```diff
@@ -1,18 +1,15 @@
...
 
-  if match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m)
-    ENV["GEM_PATH"] = ([Bundler.bundle_path.to_s] + Gem.path).join(File::PATH_SEPARATOR)
-    ENV["GEM_HOME"] = nil
-    Gem.paths = ENV
-
-    gem "spring", match[1]
-    require "spring/binstub"
+  if (match = Bundler.default_lockfile.read.match(/^GEM$.*?^    (?:  )*spring \((.*?)\)$.*?^$/m))
```
Comment (diaspora-code-review) [inline bin/spring]: Don't use parentheses around the condition of an `if`.

Comment (diaspora-code-review) [inline bin/spring]: Assignment in condition - you probably meant to use `==`.

File: bin/spring [hunk 8]
```diff
@@ -1,18 +1,15 @@
...
+    Gem.paths = { 'GEM_PATH' => [Bundler.bundle_path.to_s, *Gem.path].uniq }
```
Comment (diaspora-code-review) [inline bin/spring]: Space inside { detected.

Comment (diaspora-code-review) [inline bin/spring]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Comment (diaspora-code-review) [inline bin/spring]: Space inside } detected.

File: bin/spring [hunk 9]
```diff
@@ -1,18 +1,15 @@
...
+    gem 'spring', match[1]
```
Comment (diaspora-code-review) [inline bin/spring]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

File: bin/spring [hunk 10]
```diff
@@ -1,18 +1,15 @@
...
+    require 'spring/binstub'
```
Comment (diaspora-code-review) [inline bin/spring]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

**Comment by jhass** [comment]
@diaspora-code-review sssshhhh

File: spec/controllers/jasmine_fixtures/aspects_spec.rb [hunk 11]
```diff
@@ -75,7 +75,7 @@
       it "generates a jasmine fixture with a post containing a video", :fixture => true do
         stub_request(
           :get,
-          "http://www.youtube.com/oembed?format=json&frame=1&iframe=1&maxheight=420&maxwidth=420&scheme=https&url=http://www.youtube.com/watch?v=UYrkQL1bX4A"
+          "https://www.youtube.com/oembed?format=json&frame=1&iframe=1&maxheight=420&maxwidth=420&scheme=https&url=http://www.youtube.com/watch?v=UYrkQL1bX4A"
```
Comment (diaspora-code-review) [inline spec/controllers/jasmine_fixtures/aspects_spec.rb]: Line is too long. [158/120]

**Comment by AugierLe42e** [comment]
You can run the linters locally by typing `bin/pronto run -c develop` in your terminal.
:D

**Comment by jhass** [comment]
All files in `bin/` are auto generated.

**Comment by Flaburgan** [comment]
Such a troll >< @AugierLe42e you made me laugh very hard.

**Comment by AugierLe42e** [comment]
> Such a troll >< @AugierLe42e you made me laugh very hard.

I know, I'm sincerely sorry :p

**Comment by denschub** [comment]
Aaaaaand merged!

**Pull request closed without merging**